test/OpenTelemetry.Tests/OpenTelemetry.Tests.csproj

```diff
@@ -9,7 +9,6 @@
   <ItemGroup>
     <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry\OpenTelemetry.csproj" />
     <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry.Exporter.InMemory\OpenTelemetry.Exporter.InMemory.csproj" />
-    <ProjectReference Include="..\..\src\OpenTelemetry.Exporter.Console\OpenTelemetry.Exporter.Console.csproj" />
```

Comment:
`$(RepoRoot)\src\OpenTelemetry.Exporter.InMemory\OpenTelemetry.Exporter.Console.csproj`?

Reply:
Interestingly SDK test depend on Console exporte. Will revert this line

Reply:
@utpilla Can you check this and remove this dependency between these projects?